win_configure/msi/wix/prep.bat

```diff
@@ -0,0 +1,257 @@
+goto theend
+)
+
+echo Creating unsupported directory and moving necessory files...
+mkdir %EXECDIR%\unsupported
+if %ERRORLEVEL% == 0 (
+	For %%a in (
+		"%WINBUILDDIR%\src\unsupported\Release\*.exe"
+		"%SRCDIR%\src\unsupported\README"
+		"%SRCDIR%\src\unsupported\*.pl"
+		"%SRCDIR%\src\unsupported\*.py"
+	) do (
+	xcopy /s /d "%%~a" "%EXECDIR%\unsupported" > NUL
+	)
+) else (
+@echo Fail to create %EXECDIR%\unsupported directory.
+goto theend
+)
+
+@echo Creating scheduler privilage in home directory...
+mkdir %HOMEDIR%\sched_priv
+if %ERRORLEVEL% == 0 (
+	For %%a in (
+		"%SRCDIR%\src\scheduler\pbs_dedicated"
+		"%SRCDIR%\src\scheduler\pbs_holidays"
+		"%SRCDIR%\src\scheduler\pbs_resource_group"
+		"%SRCDIR%\src\scheduler\pbs_sched_config"
+	) do (
+	xcopy /s /d "%%~a" "%HOMEDIR%\sched_priv" > NUL
+	)
+) else (
+@echo Fail to create %HOMEDIR%\sched_priv directory.
+goto theend
+)
+
+@echo Creating lib in exec directory...
+mkdir %EXECDIR%\lib
+if %ERRORLEVEL% == 0 (
+	mkdir %EXECDIR%\lib\python\altair\pbs
+	if %ERRORLEVEL% == 0 (
+		For %%a in (
+			"%SRCDIR%\src\modules\python\pbs\*.py"
+			"%SRCDIR%\src\modules\python\pbs\*.pyo"
+			"%SRCDIR%\src\modules\python\pbs\*.pyc"			
+		) do (
+		xcopy /s /d "%%~a" "%EXECDIR%\lib\python\altair\pbs" > NUL
+		)
+	) else (
+	@echo Fail to create %EXECDIR%\lib\python\altair\pbs directory.
+	goto theend
+	)
+	
+	mkdir %EXECDIR%\lib\python\altair\pbs\v1
+	if %ERRORLEVEL% == 0 (
+		For %%a in (
+			"%SRCDIR%\src\modules\python\pbs\v1\*.py"
+			"%SRCDIR%\src\modules\python\pbs\v1\*.pyo"
+			"%SRCDIR%\src\modules\python\pbs\v1\*.pyc"	
+			"%SRCDIR%\win_configure\projects.VS2008\*.py"
+		) do (
+		xcopy /s /d "%%~a" "%EXECDIR%\lib\python\altair\pbs\v1" > NUL
+		)
+	) else (
+	@echo Fail to create %EXECDIR%\lib\python\altair\pbs\v1 directory.
+	goto theend
+	)
+
+	mkdir %EXECDIR%\lib\python\python2.7
+	if %ERRORLEVEL% == 0 (	
+		For %%a in (
+			"%binaries_path%\Python-2.7.11\Lib\*.*"			
+		) do (
+		xcopy /d "%%~a" "%EXECDIR%\lib\python\python2.7\" > NUL
+		)	
+	) else (
+	@echo Fail to create %EXECDIR%\lib\python\python2.7 directory.
+	goto theend
+	)
+	
+	For %%a in (
+			"%WINBUILDDIR%\src\lib\Libattr\Release\Libattr.lib"
+			"%WINBUILDDIR%\src\lib\Liblog\Release\Liblog.lib"
+			"%WINBUILDDIR%\src\lib\Libnet\Release\Libnet.lib"
+			"%WINBUILDDIR%\src\lib\Libpbs\Release\Libpbs.lib"
+			"%WINBUILDDIR%\src\lib\Libsite\Release\Libsite.lib"
+			"%WINBUILDDIR%\src\lib\Libwin\Release\Libwin.lib"
+		) do (
+		xcopy /s /d "%%~a" "%EXECDIR%\lib" > NUL
+		)
+		
+	mkdir %EXECDIR%\lib\ical\zoneinfo
+	if %ERRORLEVEL% == 0 (
+		xcopy /s /d "%binaries_path%\libical\zoneinfo\*.*" "%EXECDIR%\lib\ical\zoneinfo" > NUL
+	) else (
+	@echo Fail to create %EXECDIR%\lib\ical\zoneinfo directory.
+	goto theend
+	)
+
+) else (
+@echo Fail to create %EXECDIR%\lib directory.
+goto theend
+)
+
+@echo "Finished successfully"
+exit /b
+
+:theend
+@echo "Error: while preparing directory structure" 
+exit /b
```

Comment:
I am adding comment for 'license1.rtf' here as GitHub doesn't allow to put comment on binary file.
Can we rename 'license1.rtf' to something like pbs_license.rtf or pbs_license.txt ?
same for logo.ico and setup.bmp?